Makefile

```diff
@@ -161,9 +162,21 @@ $(ZKLOG_OBJDIR)/%.o: src/%.c
 	@mkdir -p $(@D)
 	gcc -o $@ -c $(ZKLOG_CFLAGS) $<
 
+.PHONY: deps
+deps $(TAP_EXEC): | $(REPO_DEPS) $(NPM_EXEC)
+	$(NPM_ENV) $(NPM) install
+
+.PHONY: ensure-node-v6-or-greater-for-test-suite
```

Comment:
I'm not sure you *need* this change. Typically this is running using the specified (currently node v8) sdcnode build. This "ensure-node-v6-or-greater-for-test-suite" guard was originally added for node packages that don't specify a particular sdcnode. It doesn't hurt tho.

Reply:
OK, I guess my inclination would be that if it's not strictly necessary then drop it.